uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -32,8 +32,7 @@ define([
       this.absencePeriods = [];
       this.absenceTypes = [];
       this.calendar = {};
-      //1. the contact is selected by manager when applying absence on behalf of manager
-      //2. the contact is set when manager is responding to request to display name
+      //1. the contact is set when manager is responding to request to display name
       this.contact = null;
```

Comment:
I believe we agreed to change the name of this property to something more representative of it's use, like `contactName`